test/test_geocaching.py

```diff
@@ -129,12 +129,26 @@ def test_search_rect(self):
                 distances = [great_circle(cache.location, origin).meters for cache in caches]
                 self.assertEqual(distances, sorted(distances))
 
-    def test_api_rate_limit(self):
+    def test__try_getting_cache_from_guid(self):
+        # get "normal" cache from guidpage
+        with self.recorder.use_cassette('geocaching_shortcut_getcache__by_guid'):  # is a replacement for login
+            cache = self.gc._try_getting_cache_from_guid('15ad3a3d-92c1-4f7c-b273-60937bcc2072')
+            self.assertEqual("Nekonecne ticho", cache.name)
+
+        # get PMonly cache from GC code (doesn't load any information)
+        cache_pm = self.gc._try_getting_cache_from_guid('328927c1-aa8c-4e0d-bf59-31f1ce44d990')
+        cache_pm.load_quick()  # necessary to get name for PMonly cache
+        self.assertEqual("Nidda: jenseits der Rennstrecke Reloaded", cache_pm.name)
+
+
+class TestAPIRateLimiting(NetworkedTest):
```

Comment:
Shouldn't we use the usual naming scheme here, so `TestApiRateLimiting`?

Reply:
https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles

> Note: When using acronyms in CapWords, capitalize all the letters of the acronym. Thus HTTPServerError is better than HttpServerError.

Reply:
Okay, did not know this ;)